src/parameterizations/lateral/MOM_hor_visc.F90

```diff
@@ -975,6 +1048,15 @@ subroutine hor_visc_init(Time, G, param_file, diag, CS)
                  "often 0.15.", units="nondim", default=0.0, &
                   fail_if_missing = CS%Smagorinsky_Kh)
 
+    call get_param(param_file, mod, "LEITH_KH", CS%Leith_Kh, &
+                 "If true, use a Leith nonlinear eddy viscosity.", &
+                 default=.false.)
+    if (CS%Leith_Kh .or. get_all) &
+      call get_param(param_file, mod, "LEITH_LAP_CONST", Leith_Lap_const, &
+                 "The nondimensional Laplacian Leith constant, \n"//&
+                 "often ??", units="nondim", default=0.0, &
```

Comment:
?? (referring to symbols in log message). If there isn't a typical value to recommend then let's remove the sentence from the documentation.